python/ray/tune/examples/pbt_convnet_example.py

```diff
@@ -52,11 +52,11 @@ def _restore(self, checkpoint_path):
         self.model.load_state_dict(torch.load(checkpoint_path))
 
     def reset_config(self, new_config):
-        del self.optimizer
-        self.optimizer = optim.SGD(
-            self.model.parameters(),
-            lr=new_config.get("lr", 0.01),
-            momentum=new_config.get("momentum", 0.9))
+        for param_group in self.optimizer.param_groups:
+            param_group["lr"] = new_config.get("lr")
```

Comment:
what if the new_config does not have `lr` or `momentum`?

Reply:
Do you mean we should use new_config.get("lr", 0.01)? Sure we can always use that. Just want to learn when will new_config does not have lr or momentum. Thanks.

Reply:
One situation is if the user does not plan to tune momentum, for example

Reply:
Oh, Do you mean users might want to change the code of this example and try to tune by themselves? Sure. I'll update it with a default value.